python/lsst/analysis/tools/actions/plot/rhoStatisticsPlot.py

```diff
@@ -39,6 +39,8 @@
 
 
 class RhoStatisticsPlotAction(PlotAction):
+    """A class to plot rho statistics."""
+
```

Comment:
Arun has just fixed a few of these, just make sure you don't overwrite them on rebase.